.sync.yml

```diff
@@ -28,6 +28,11 @@ spec/spec_helper.rb:
 Rakefile:
   changelog_user: "puppetlabs"
 
+spec/default_facts.yml:
+  extra_facts:
+    pe_server_version: '2016.4.5'
```

Comment:
Lets update this to a recent version.

Reply:
I had done this, but might have undone it with a lazy rebase.  I'll change it again and rerun the tests.